kuma/users/migrations/0001_squashed_0008_update_locales_tz_help.py

```diff
@@ -49,9 +49,9 @@ class Migration(migrations.Migration):
                 ('linkedin_url', models.TextField(blank=True, verbose_name='LinkedIn', validators=[django.core.validators.RegexValidator(b'^https?://((www|\\w\\w)\\.)?linkedin.com/((in/[^/]+/?)|(pub/[^/]+/((\\w|\\d)+/?){3}))$', 'Enter a valid LinkedIn URL.', b'invalid')])),
                 ('facebook_url', models.TextField(blank=True, verbose_name='Facebook', validators=[django.core.validators.RegexValidator(b'^https?://www\\.facebook\\.com/', 'Enter a valid Facebook URL.', b'invalid')])),
                 ('stackoverflow_url', models.TextField(blank=True, verbose_name='Stack Overflow', validators=[django.core.validators.RegexValidator(b'^https?://stackoverflow\\.com/users/', 'Enter a valid Stack Overflow URL.', b'invalid')])),
-                ('groups', models.ManyToManyField(related_query_name='user', related_name='user_set', to=b'auth.Group', blank=True, help_text='The groups this user belongs to. A user will get all permissions granted to each of his/her group.', verbose_name='groups')),
+                ('groups', models.ManyToManyField(related_query_name='user', related_name='user_set', to='auth.Group', blank=True, help_text='The groups this user belongs to. A user will get all permissions granted to each of his/her group.', verbose_name='groups')),
```

Comment:
I wasn't expecting the migration to change. I wonder if this should instead be a new migration. We probably have a few things that will change as bytes are turned into strings, such as choice fields.